src/main.ts

```diff
@@ -2,7 +2,7 @@
 
 import './operators';
 export * from './angular-cesium.module';
-export * from './models';
-export * from './components';
-export * from './services';
+export * from './models/index';
```

Comment:
Let's take an example - models.
What angular-seed did is - searching for models.js. 
@ttsahi @shiloDrot  are my witnesses.

That's why, in order to avoid those problems I just added index so it will try fetching models/index.js